### PR TITLE
Post-release version bump

### PR DESCRIPTION
**Pull request by alexweininger**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 
-## Unreleased
+## 0.23.2 - 2022-04-13
+### Fixed
+- Zip deploy fails with ECONNRESET or 400 Bad Request on VS Code versions `>=1.66.0` [#2191](https://github.com/microsoft/vscode-azureappservice/issues/2191)
 
 ## 0.23.1 - 2022-02-24
 ### Changed
 - Adopt VS Code's new [proposal API guidelines](https://github.com/microsoft/vscode/blob/fc7fb5d480418d149ee226ebd45e9c590e240cb5/src/vscode-dts/README.md)
-- Minimum version of VS Code is now 1.63.0
```
Comment (wwlorey) [inline CHANGELOG.md:8]: Why was this line removed?

**Review by wwlorey — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (alexweininger) [reply CHANGELOG.md]: 😱  thank you for catching this

**Review by wwlorey — APPROVED** [review]
